bootloader/script_test.go

```diff
@@ -0,0 +1,118 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2014-2015 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package bootloader_test
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	. "gopkg.in/check.v1"
+
+	"github.com/snapcore/snapd/bootloader"
+	"github.com/snapcore/snapd/bootloader/assets"
+)
+
+type scriptTestSuite struct {
+	baseBootenvTestSuite
+
+	bootdir string
+}
+
+var _ = Suite(&scriptTestSuite{})
+
+func (s *scriptTestSuite) TestTrivialFromScript(c *C) {
+	e, err := bootloader.EditionFromScript(bytes.NewBufferString(`# X-Snapd-boot-script-edition: 321
```

Comment:
could you have a unit test with this in the script multiple times, i.e.

```
# X-Snapd-boot-script-edition: 321
next line
one after that
# X-Snapd-boot-script-edition: 3333
```

and 

```
# X-Snapd-boot-script-edition: 321
# X-Snapd-boot-script-edition: 3333
next line
one after that
```

Reply:
Added now.